### DISCUSSION TITLE
dockerfile for graphql-engine

### DISCUSSION THREAD

**Discussion by rdinnik** [hasura/graphql-engine#8851] [Question] [answered]
Where can we find an example of graphql-engine Dockerfile for published images?

hasura/graphql-engine
> v.1.3.X
v.2.0.0

thank you,
Roman

**Comment by jberryman** [comment, marked as answer]
We don't have a self-contained dockerfile that builds and packages unfortunately, but it should be pretty straightforward to do this yourself:

- look at `.circleci/server-builder.dockerfile`; this has all the dependencies to build the project
- take a look at CI scripts or `dev.sh` for how to build 
- `server/packaging/build/Dockerfile` for packaging
